### PR TITLE
Added incremented logging to neard and rotating logs to nearup (https://github.com/near/nearup/issues/124)

### PR DESCRIPTION
**Pull request by chefsale**
```
sandi@sandi-ThinkPad-X1-Carbon-7th ~/w/n/nearup (logs-increase)> ls ~/.nearup/logs/
betanet.log  nearup.log  watcher.log
sandi@sandi-ThinkPad-X1-Carbon-7th ~/w/n/nearup (logs-increase)> nearup run betanet
2020-10-09 14:06:15.324 INFO nearup - run: Home directory is /home/sandi/.near/betanet...
2020-10-09 14:06:15.324 INFO nodelib - setup_and_run: Using officially compiled binary
2020-10-09 14:06:17.488 INFO util - download_binaries: Downloading latest deployed version for betanet
2020-10-09 14:06:17.489 INFO util - download_binaries: Downloading near to /home/sandi/.nearup/near/betanet/near from nearcore/Linux/1.16.0/404bc6bf6349803ac10c55f5a51eae7ac3895310/near...
2020-10-09 14:06:21.505 INFO util - download_binaries: Downloaded near to /home/sandi/.nearup/near/betanet/near...
2020-10-09 14:06:21.506 INFO util - download_binaries: Making the near executable...
2020-10-09 14:06:21.506 INFO util - download_binaries: Downloading genesis-csv-to-json to /home/sandi/.nearup/near/betanet/genesis-csv-to-json from nearcore/Linux/1.16.0/404bc6bf6349803ac10c55f5a51eae7ac3895310/genesis-csv-to-json...
2020-10-09 14:06:24.585 INFO util - download_binaries: Downloaded genesis-csv-to-json to /home/sandi/.nearup/near/betanet/genesis-csv-to-json...
2020-10-09 14:06:24.586 INFO util - download_binaries: Making the genesis-csv-to-json executable...
2020-10-09 14:06:27.002 INFO nodelib - genesis_changed: The genesis version is up to date
2020-10-09 14:06:27.017 INFO nodelib - run_binary: /home/sandi/.nearup/logs/betanet
2020-10-09 14:06:27.017 INFO nodelib - run_binary: /home/sandi/.nearup/logs/betanet.log
2020-10-09 14:06:27.017 INFO nodelib - run_binary: <function next_logname at 0x7f6a228a5670>
2020-10-09 14:06:27.028 INFO nodelib - run: Node is running...
2020-10-09 14:06:27.028 INFO nodelib - run: To check logs call: `nearup logs` or `nearup logs --follow`
2020-10-09 14:06:27.028 INFO nodelib - run: Watcher is enabled. Starting watcher...
2020-10-09 14:06:27.028 INFO watcher - run_watcher: Starting the nearup watcher...
sandi@sandi-ThinkPad-X1-Carbon-7th ~/w/n/nearup (logs-increase)> ls ~/.nearup/logs/
betanet.log  betanet.log.1  nearup.log  watcher.log
```

### CODE REVIEW

**Comment by nickmonad** [comment]
Small tip: You can write "Resolves #124" in the PR description and it will auto-link this PR to that issue and close it when merged :)

File: nearup [hunk 1]
```diff
@@ -19,7 +21,11 @@ logging.basicConfig(
     '%(asctime)s.%(msecs)03d %(levelname)s %(module)s - %(funcName)s: %(message)s',
     datefmt='%Y-%m-%d %H:%M:%S',
     handlers=[
-        logging.FileHandler(os.path.expanduser('~/.nearup/logs/nearup.log')),
+        handlers.RotatingFileHandler(
+            os.path.expanduser('~/.nearup/logs/nearup.log'),
+            maxBytes=1024 * 1024,
+            backupCount=10,
```
Comment (nickmonad) [inline nearup:27]: what does `backupCount` mean in this context?

Comment (chefsale) [reply nearup:27]: for logs to be log.1 log.2 ... log.10

Comment (nickmonad) [reply nearup:27]: Does that mean we could only restart 10 times?

Comment (chefsale) [reply nearup:27]: No after that starts circularly I believe

**Review by nickmonad — APPROVED** [review]

**Pull request merged**